lib/krane/cluster_resource_discovery.rb

```diff
@@ -17,7 +17,7 @@ def crds
     end
 
     def prunable_resources(namespaced:)
-      black_list = %w(Namespace Node)
+      black_list = %w(Namespace Node ControllerRevision)
```

Comment:
`ControllerRevision` are created by DS controllers and are use for rollbacks. They get the last-applied annotation and I suspect we dont' want to be pruning them https://kubernetes.io/docs/tasks/manage-daemon/rollback-daemon-set/#understanding-daemonset-revisions.

Reply:
Good catch